test_cli.py

```diff
@@ -1675,8 +1675,8 @@ def test(ctx):
     run_command([cl, 'wedit', wuuid, '--title', 'nonunicode'])
     check_contains('nonunicode', run_command([cl, 'print', wuuid]))
 
-    # TODO: enable with Unicode support.
-    run_command([cl, 'wedit', wuuid, '--title', 'fáncy ünicode 你好世界😊'], 1)
+    # unicode in worksheet title
+    run_command([cl, 'wedit', wuuid, '--title', 'fáncy ünicode 你好世界😊'], 0)
     # check_contains('fáncy ünicode 你好世界😊', run_command([cl, 'print']))
```

Comment:
@yipenghe You didn't uncomment out the second part of the test, which actually checks that the title is shown as unicode. Can you uncomment this line and make a PR?

Reply:
Good catch, will do! Thanks!